src/lib/application/files/ts/tsconfig.json

```diff
@@ -11,6 +11,11 @@
     "outDir": "./dist",
     "baseUrl": "./",
     "incremental": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "strictNullChecks": <%= strict %>,
+    "noImplicitAny": <%= strict %>,
```

Comment:
I'm not sure if I should keep all this options here, or create another tsconfig template for the strict mode.

Reply:
I like the idea of keeping these to help new devs know the options exist